README.md

```diff
@@ -227,6 +233,48 @@ const { default: Carousel, Dots } = dynamic(
 );
 ```
 
+### Renovate
+We've decided that [Renovate](https://github.com/renovatebot/renovate) is better than [Greenkeeper](https://github.com/greenkeeperio/greenkeeper) because Renovate is very configurable and has a great support.
+
+### IssueHunt
+We've decided to use [IssueHunt](https://issuehunt.io/) to fund issues so we can get more contributors (more contributors, more popular a given project), assign a value to issues and reward active contributors.
+
+### Testing environment deployment
+We've decided to use http://beghp.github.io/ domain to deploy each branch there because in a version deployed to [Netlify](https://www.netlify.com/) we've noticed broken fonts so deploying to GitHub Pages gives an environment the most possibly similar to the [production version](https://brainhubeu.github.io/react-carousel/). `beghp` is an acronym from Brainhub.eu GitHub Pages and we use this organization in order to keep only real repos in the `brainhubeu` organization.
+
+### Labels
```

Comment:
In my opinion that should be a part of Contributing header

Reply:
@humbak 

fixed in https://github.com/brainhubeu/react-carousel/pull/482/commits/6b8cf6598366c15d53ef737a15258e0276fbfaaa